graylog2-web-interface/src/logic/rest/FetchProvider.js

```diff
@@ -78,15 +80,15 @@ export class Builder {
 
         // Redirect to the start page if a user is logged in but not allowed to access a certain HTTP API.
         if (SessionStore.isLoggedIn() && error.status === 403) {
-          ErrorsActions.report(createUnauthorizedError(error));
+          ErrorsActions.report(createFromFetchError(fetchError));
```

Comment:
While having another look, I realised we were not providing a real `FetchError` when creating a `ReportedError`.
I've decided to create a `FetchError` instead of using the `SuperagentPromiseError`, because the checks we are doing when we create a `FetchError` are still useful and it is easier to work with our own interface.

Reply:
One side note regarding the `FetchProvider`, during the development we've discussed the unification of redundant code. I did not implemented these changes inside this PR, because they are also part of the CSV export PR https://github.com/Graylog2/graylog2-server/pull/7709